src/frontend/main.go

```diff
@@ -31,6 +31,18 @@ import (
 	"go.opencensus.io/plugin/ochttp/propagation/b3"
 	"go.opencensus.io/stats/view"
 	"go.opencensus.io/trace"
+
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
```

Comment:
Worth adding a comment to explain which package are you using (exporter directly vs. collector and then exporting it)

Reply:
done